src/libxtp/gwbse/gw.cc

```diff
@@ -328,18 +324,21 @@ double GW::SolveQP_Bisection(double lowerbound, double f_lowerbound,
   while (true) {
     double c = 0.5 * (lowerbound + upperbound);
     if (std::abs(upperbound - lowerbound) < _opt.g_sc_limit) {
```

Comment:
In the first if statement, the x-value is compared to the tolerance. In the second if statement, the y-value is compared to the tolerance. So, inconsistent units for `_opt.g_sc_limit`!

Reply:
wikipedia des it the same way and we only want the frequency to be accurate to `_opt.g_sc_limit`